lighthouse-core/config/config.js

```diff
@@ -145,7 +146,7 @@ function validateCategories(categories, audits, groups) {
     return;
   }
 
-  const auditIds = audits.map(audit => audit.meta.name);
+  const auditIds = audits.map(audit => audit.implementation.meta.name);
```

Comment:
I find `implementation` such a weird name I wanted to call it instance but I see below that it's indeed the implementation but still find it weird 😛 but maybe that's because i'm no native speaker.